ngraph/core/include/ngraph/op/range.hpp

```diff
@@ -43,6 +43,9 @@ namespace ngraph
                 bool evaluate(const HostTensorVector& outputs,
                               const HostTensorVector& inputs) const override;
                 bool has_evaluate() const override;
+                void set_output_type(element::Type output_type) { m_output_type = output_type; }
```

Comment:
Is it expected that this method won't work with the pointer to `ngraph::Node`?
And this code will have different behavior:
```
ngraph::op::v4::Range* range;
ngraph::Node* node = range;
// Result range->set_output_type(output_type) != node->set_output_type(0, output_type, shape)
```
Maybe can we use the different name?

Reply:
it's common approach with v3::ShapeOf -- i simply used the same logic